lib/jsurl.js

```diff
@@ -45,17 +47,26 @@
 			case 'object':
 				if (!v) return '~null';
 				if (Array.isArray(v)) {
-					return '~(' + (v.map(function(elt) {
-						return stringify(elt) || '~null';
-					}).join('') || '~') + ')';
+					for (var i = 0; i < v.length; i++) {
+						if (i in v) {
```

Comment:
This test should be removed because `JSON.stringify` outputs a null for missing array entries:

```js
var foo = []
foo[1] = 11;
foo[3] = 33;
console.log(JSON.stringify(foo));
```
Output:
```
[null,11,null,33]
```